src/components/RoomView/Card.jsx

```diff
@@ -66,8 +66,8 @@ const Card = ({ word }) => {
         update(gameRef, {
           team1RemainingCards: teamOneRemainingCards - 1,
         });
-        // decrement from guesses remaining from spymasters clue
-        // if guesses remaining === 0, endTurn()
+        set(child(gameRef, 'guessesRemaining'), guessesRemaining - 1);
+        // set(guessesRemainingRef, guessesRemaining - 1);
```

Comment:
can you give a note on what the commented out version would do / why you chose to leave it here, just commented out?